xdesign/__init__.py

```diff
@@ -56,6 +56,8 @@
 from xdesign.metrics import *
 from xdesign.plot import *
 from xdesign.material import *
+from xdesign.propagation import *
+from xdesign.grid import *
```

Comment:
Changelog says you removed the grid module, but here is an import statement for it. Ooops!